### ISSUE TITLE
How do I update arpl?

### ISSUE THREAD

**Issue by lingyinsam** [fbelavenuto/arpl#671]
The version of arpl I use is old. I want to update arpl perfectly. How do I go about it?

**Comment by lingyinsam** [comment]
The current arpl version number seems to be v0.3beta3.

**Comment by lingyinsam** [comment]
> 不需要更新，如果没有任何问题，如果你想更新到v1.1，你应该刷新你的删除磁盘
I don't know what you mean. Does that mean that if I want to upgrade to v1.1 I need to redo a boot disk? Or do you still need to reinstall your system?

**Comment by Brian099** [comment]
redo, I remember since one version, it changed too much, arpl need redo your loader disk.

**Comment by dasb00ter** [comment]
Was wondering if somebody might do a write up on updating arpl and then updating DSM. I tried asking in the xpenology forum but it is often filled with more technical questions. 
I am not even sure it is necessary to update the loader before trying to update DSM?
It seems that part of the power of the loaders design is its updatability and although it is intuitive the consequences of doing wrong seem dire. 
In short an explanation of updating ARPL(if that is recommended) then updating DSM somewhere on the internet would be great

**Comment by wjz304** [comment]
"Update menu"-"Update arpl"
